gdraw/gtextfield.c

```diff
@@ -916,6 +916,11 @@ static void GTextFieldSave(GTextField *gt,int utf8) {
     FILE *file;
     unichar_t *pt;
 
+    GEvent e;
+    e.type = et_save;
+    e.w = gt->g.base;
+    GDrawPostEvent(&e);
```

Comment:
Move this to the very end of the function. Also look at how `GTextFieldChanged` implements the send, and copy that (making et_save a subtype of et_controlevent)